docs/synapse/userguides/data_model.rstorm

```diff
@@ -0,0 +1,555 @@
+to maintain the distinction between the **template** for creating an object (a form) and an **instance** of a particular
+object (a node). ``inet:fqdn`` is a form; ``inet:fqdn = vertex.link`` is a node.
+
+All forms must have a **primary property.** The primary property is the name of the form and the definition of the
+value to be provided for individual instances (nodes) of that form. The primary property must be defined so that it
+is unique across all possible instances of that form. For example, FQDNs are unique, based on the way they are defined
+and registered (two different organizations cannot both register the FQDN ``vertex.link``). So the primary property
+value of an ``inet:fqdn`` is simply the FQDN itself.
+
+All properties in Synapse must have a defined **type;** in many cases, a form is also its own type (for example, the form
+``inet:fqdn`` has a type of ``inet:fqdn``).
+
+Forms may have **secondary properties** that record additional information about the form or further describe it.
+Secondary properties are form-specific and are explicitly defined for each form.
+
+Synapse also supports a set of universal secondary properties (**universal properties**) that are valid for all forms.
+
+.. _form-namespace:
+
+Form Namespace
+++++++++++++++
+
+Synapse uses a structured namespace for forms. Each form name consists of at least two elements separated by a
+colon ( ``:`` ). For example:
+
+ - ``file:bytes``
+ - ``inet:fqdn``
+ - ``ou:org``
+ - ``risk:threat``
+ 
+The first element in the namespace represents a rough "category" for the form (i.e., ``inet`` for Internet-related
+objects). The Synapse data model is broad and extensible. The ability to group portions of the data model into
+related categories makes a large model easier to manage, and also allows Synapse users to focus on those portions
+of the model most relevant to them.
+
+The second and / or subsequent elements in the form name define the specific "subcategory" or "thing" within the
+form's primary category (e.g., ``inet:fqdn`` represents a fully qualified domain name (FQDN) within the "Internet"
+(``inet``) category.
+
+**Properties** have a namespace that extends the form namespace (form names are also primary properties). See
+:ref:`data-props` and :ref:`prop-namespace` below for additional detail.
+
+.. _data-node:
+
+Node
+----
+
+A **node** is a unique object within Synapse; they are specific instances of generic forms. Every node consists of:
+
+- A **primary property**, represented by the form of the node plus its value (``<form> = <valu>``). All primary
+  properties must be unique for a given form; the uniqueness of the ``<form> = <valu>`` pair ensures there can 
+  be only one node in Synapse that represents the domain ``woot.com``.
+  
+  .. TIP::
+    
+    Nodes are **deconflicted** based on their primary property to ensure (as far as possible) that duplicate
+    nodes (that represent the same "thing") cannot exist within a single Cortex.
```

Comment:
This is kind of an odd line; since its possible for the same node to exist across multiple distinct views; but the nuance of that is beyond the scope of this document.

Reply:
Fair point. The "deconfliction" statement is not really necessary so will cut.